download-build-run-locale-screenshots.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+HOME=""
```

Comment:
also add a 1-2 line description of what this script is for, and how it can be used

Reply:
Also, need to specify the prerequistes for running this script (i.e. installing fastlane, screengrab, etc.)